src/pages/_app.tsx

```diff
@@ -36,16 +36,9 @@ import '../styles/global.scss';
 import nextI18nextConfig from '../../next-i18next.config';
 import { api } from '~/utils/api';
 
-function App(
-  this: any,
-  props: AppProps<{
-    colorScheme: ColorScheme;
-    packageAttributes: ServerSidePackageAttributesType;
-    editModeEnabled: boolean;
-    defaultColorScheme: ColorScheme;
-  }>
-) {
+function App(props: AppProps & { colorScheme: ColorScheme }) {
```

Comment:
Why the colorScheme, packageAttributes, editModeEnabled and defaultColorScheme has been removed?

Reply:
Because they are not used in the code, previously `props.pageProps.{item}` was used and it didn't change anything

Reply:
![image](https://github.com/ajnart/homarr/assets/63781622/78c50812-6295-4719-b375-aeec8f6e0421)
Yeah because it is any now 😂

Reply:
Can you please revert this to the previous one?

Reply:
this

Reply:
> ![image](https://github.com/ajnart/homarr/assets/63781622/78c50812-6295-4719-b375-aeec8f6e0421)
> 
> Yeah because it is any now 😂 

PageProps was always any, it’s directly props that was typed, so it doesn’t really change anything. But I see your point I’ll correct this issue

Reply:
No I fixed that ones:
![image](https://github.com/ajnart/homarr/assets/63781622/381cd270-1c5f-423f-8d60-71fcf604257e)

Reply:
There you go I reverted it. I don't know why I was confused lol